config/locales/en.yml

```diff
@@ -54,16 +54,28 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      abuse: "contact our Abuse Team"
+      sincerely: "Sincerely,"
+      ao3_abuse: "AO3 Abuse"
+      part2: "We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it."
+      html:
+        part1: "Your work <i>%{title}</i> has been reported as being in violation of the Archive of Our Own's %{tos}."
+        part3: "If you have any questions, please contact our Abuse Team (%{abuse})"
```

Comment:
`part3` for the HTML email here says `please contact our Abuse Team (%{abuse})` but line 61 is `abuse: "contact our Abuse Team"`. 

So would the HTML email end up saying `please contact our Abuse Team (<a href="url for abuse">contact our Abuse Team</a>)` instead of the desired `please <a href="url for Abuse">contact our Abuse Team</a>`?